_posts/2020-04-18-regression-chains/regression-chains.Rmd

```diff
@@ -0,0 +1,179 @@
+---
+title: "Regression chains"
+categories:
+  - regression
+  - mlr3pipelines
+author:
+  - name: Lennart Schneider
+date: 04-18-2020
+description: |
+  We show how to use mlr3pipelines to do regression chains.
+output:
+  distill::distill_article:
+    self_contained: false
+    css: ../../custom.css
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  R.options = list(width = 80)
+)
+library(mlr3book)
+lgr::get_logger("mlr3")$set_threshold("warn")
+```
+
+In this tutorial we demonstrate how to use `r mlr_pkg("mlr3pipelines")` to handle multi-target regression by arranging regression models into a chain, i.e., creating a linear sequence of regression models.
+
+## Regression Chains
+
+In a simple regression chain, regression models are arranged in a linear sequence. Here, the first model will use the input to predict a single output and the second model will use the input and the output of the first model to make its own prediction and so on. For more details, see e.g. [Spyromitros-Xioufis, Tsoumakas, Groves & Vlahavas (2016)](https://doi.org/10.1007/s10994-016-5546-z).
+
+## Before you start
+
+The following sections describe an approach towards working with `Task`s that have multiple targets.
+E.g. in the example below, we have 3 target variables **y1** to **y3**.
+This type of Task can be created via the [mlr3multioutput](https://github.com/mlr-org/mlr3multioutput/) (currently under development) in the future.
+The current goal of this post is to show how such modeling steps can be written as a relatively small amount of pipeline steps and how such steps can be put together. Writing pipelines with such steps allows for great flexibility in modelling more complicated scenarios such as the ones described below.
```

Comment:
Add a sentence: In the future, [mlr3multioutput](https://github.com/mlr-org/mlr3multioutput/) will offer simple chaining approaches as pre-built pipelines (reference ppl), while the steps outlined below outline the basic approach behind them, while offering full flexibility for further customisation.